setup.py

```diff
@@ -85,8 +115,11 @@ class BuildExt(build_ext):
         'unix': [],
     }
 
+    UnixCCompiler.src_extensions.append(".mm")
+    UnixCCompiler.language_map[".mm"] = "objc"
+
     if sys.platform == 'darwin':
-        darwin_opts = ['-stdlib=libc++', '-mmacosx-version-min=10.7']
+        darwin_opts = ['-stdlib=libc++', '-std=gnu++14']
```

Comment:
I believe this comes from the macOS obs plugin CMake file; probably not necessary?

Reply:
Fixed in https://github.com/letmaik/pyvirtualcam/pull/22.